4C Integration.md

```diff
@@ -0,0 +1,262 @@
+# 4C integration doc
+
+###### This document will give a brief overview of the MX media-buying platform, and explain how to interact with its API.
+
+### Platform Overview
+
+#### Domain Concepts
+
+Within the MX platform we think in terms of `Organizations`, `Users`, `Teams`, `Roles`, `Markets`, `Orders`, `Assets`, `Attributes`, and `Matches`.
+
+* An `Organization` represents a single buyer or seller, and is analogous to a company account. There are many `Users` within an `Organization`, and any action a user takes looks from the outside as if it was done by the organization. Users are assigned to `Teams`, and given `Roles` within those teams which constrain which actions they are allowed to take.
+
+* A `Market` is an environment in which transactions happen. There is an `Open Market` and several `Private Markets`. Initially, all transactions will happen in the latter. `Orders` in private markets can only be seen by market members, which are `Organizations`, and defined by the market owner.
+
+* An `Order` is how we represent supply or demand entered into a `Market`. An order represents the intention of an `Organization` to buy/sell some amount of a certain `Asset` in a `Market`, at some price.
+
+* An `Asset` is what we call media that is bought and sold. It is composed of `Attributes` and additional information, like the date and time. An `Attribute` represents some piece of information about the asset. `Attributes` in turn have `Types`, which can be thought of as classifications of information about assets. This is displayed in the UI in the format of `Type: Attribute Value`, ie, `Network: AMC`.
+
+* A `Match` is the outcome of a market transaction; it represents an intersection of supply and demand, on terms meeting the requirements of both sides. Matches are analogous to non-preemptible orders in traditional media buys.
+
+#### Components
+
+The buy-side MX platform consists of several areas of functionality; `Organization Management`, the `Market`, and `Brokerage`. There are others, but these are all that is relevant for this implementation.
+
+* Organization Management
+    * Profile
+    * Settings
+    * Permissions
+* Market
+    * Query (aka View)
+    * *Watch (Planned)*
+    * Private
+* Brokerage
+    * Campaigns
+    * Active
+    * History
+    * Log
+
+### Market
+
+#### Rules
+
+* The market operates on **New York time (EDT, GMT-5)**, and is open from **9:30AM-4:30PM**. Orders may be submitted at any time, but will **only be processed** when the market is open.
+
+* Orders are processed in a **first-in-first-out** queue; if your order is submitted before another one, it is guaranteed to get access to supply first. If an order is edited, its position in the queue is reset.
```

Comment:
maybe 'If an order is edited, it becomes last in queue' instead? 'reset' seems a little ambiguous